tasks/fix-cat2.yml

```diff
@@ -1511,12 +1552,15 @@
       - RHEL-07-040740
 
 - name: "MEDIUM | RHEL-07-040810 | PATCH | The system must use a local firewall."
```

Comment:
Yeah, was actually curious if we should leave it now and remove it when the final rev is complete? Not sure which one they will remove. Figured once the final release is out, do a final check for the actual findings, add those that are missing, and remove those that are taken out. 

Thoughts?